swan-cern/files/swan_computing_config.py

```diff
@@ -191,6 +271,12 @@ def _modify_containers_for_spark(self, hadoop_secret_name):
             ),
         )
 
+    def _gpu_enabled(self):
+        """
+        return True if the user has requested a GPU
+        """
+        return "cu" in self.spawner.user_options[self.spawner.lcg_rel_field]
```

Comment:
Depending on how Rodrigo implements the custom env in the spawner, this might crash ^

Reply:
If so, he'll fix it :) This is purely a code move.